experimenter/manifesttool/tests/test_version.py

```diff
@@ -411,9 +411,9 @@ def test_parse_version_file_plist(self, version_str, expected):
             (
                 app_config,
                 [
-                    (Ref("v120", "r0"), "120.0a1\n"),
-                    (Ref("v120.1", "r1"), "120.1a1\n"),
-                    (Ref("v121", "r2"), "121.0a1\n"),
+                    (Ref("v120", "r0"), b"120.0a1\n"),
+                    (Ref("v120.1", "r1"), b"120.1a1\n"),
+                    (Ref("v121", "r2"), b"121.0a1\n"),
```

Comment:
Why do we need these newlines?

Reply:
Those are the bodies of the mock HTTP responses. We don't technically need them, but version files in practice all end with a newline (e.g., https://github.com/mozilla-mobile/firefox-android/blob/main/version.txt)